Cafepresscart

Actiondan2857 opened the pull request.

timrobuck commented:
Dan, you have some conflicts to resolve before it can merge.
Conflicting files
Autodan/src/pages/cafepress.com/CpCartPageObject.cs 
Autodan/src/pages/cafepress.com/CpCommonPageObject.cs 
Autodan/src/pages/cafepress.com/CpProductDescriptionPageObject.cs 

other than that, just a few comments you perhaps did not intend to leave in. 
//BtnSaveShoppingCartSendLink.Click();
//BtnEstShippingOptionsCalculate.Click();
//CartAdjustQuantityTest();

Actiondan2857 commented:
added todo: comments to the three click actions you outlined.

They were intentionally commented out; I just did a poor job of adding an actual comment! haha

Those actions are part of scope for the test; but in the current dev environment for cart- those actions all either crash the page or cause other errors.
I added todo comments explaining each one, and includes a note that they will need to be updated once we get our new environment stood up.
